wasp-ai/src/client/pages/MainPage.jsx

```diff
@@ -78,7 +78,7 @@ const MainPage = () => {
         localStorage.setItem("appNum", 0);
       }
```

Comment:
The logic above, where appNum is obtained from local storage and set to 0 if not defined, could be encapsulated into a function `getAppNum()` that would hide the details of how it is done.

Btw the logic is a bit weird here -> you fetch and parse appNum, but if it is 0, you set it to 0 in local storage -> so you are not just fetching, but also setting here, but only in a special case, a bit weird.
I think it is better to get it and that is it. If it is unset, you return 0 while getting it, but don't try to set it to 0 -> no need, if it is undefined we treat it as 0 anyway.